jobs/ftp-poller/invoke_jobs.py

```diff
@@ -64,16 +64,16 @@ def shell_context():
 
 
 def run(job_name):
-    from tasks.poll_ftp_task import PollFtpTask
+    from tasks.cas_poller_ftp import CASPollerFtpTask
 
     application = create_app()
 
     application.app_context().push()
     if job_name == 'FTP_POLLER':
-        PollFtpTask.poll_ftp()
+        CASPollerFtpTask.poll_ftp()
         application.logger.info(f'<<<< Completed Polling FTP >>>>')
     else:
-        PollFtpTask.poll_ftp()
+        CASPollerFtpTask.poll_ftp()
```

Comment:
Is this correct, or should it be cgi poller ftp ?

Reply:
Corrected it.Thanks for catching it